doc/Type/Array.pod6

```diff
@@ -22,16 +22,16 @@ called on multi-dimensional arrays.
 Exactly the same as L«C<List.gist>|/type/List#method_gist»,
 except using square brackets for surrounding delimiters.
 
-=head2 routine pop
+=head2 method pop
 
 Defined as:
 
-    multi sub    pop(Array:D )
-    multi method pop(Array:D:)
+    method pop(Array:D:) is nodal
 
-Removes and returns the last item from the array.  Fails for an empty array.
+Removes and returns the last item from the array.  Fails if the array is empty.
 
-Example:
+Like many C<Array> methods, method C<pop> may be called via the corresponding
+L<subroutine|/language/independent-routines#sub_pop>. For example:
 
     my @foo = <a b>; # a b
     @foo.pop;        # b
```

Comment:
Right next to this line is the `pop @foo` example statement. You haven't removed it from here... and moved it to the new placement. It might be a good idea to leave it here, but then it would also be convenient to link the new reference for it.

Reply:
The reference is in line 34 :)

Reply:
Ah, right sorry. Still, what about the example?

Reply:
You wrote it might be a good idea to leave it here, and I agree :-)

Reply:
I'm not totally sure that's what I wanted to write then, but point taken.